dbt-metricflow/pyproject.toml

```diff
@@ -50,6 +50,9 @@ redshift = [
 snowflake = [
   "dbt-snowflake~=1.6.0"
 ]
+trino = [
+  "dbt-trino~=1.6.0"
```

Comment:
Yeah, i am not sure why github diff is not showing diff with recent changes. However, If you click on show changes from last commit from files changed page you can see those 
![image](https://github.com/dbt-labs/metricflow/assets/138710149/ef6f0d11-4f91-47e4-b3fa-92a070b0f9f8)
Looks like there is discussion about [this](https://github.com/orgs/community/discussions/16351)

Reply:
I think this PR has a different problem, as it's rendering the branch commit diffs correctly:

https://github.com/dbt-labs/metricflow/compare/main...sarbmeetka:metricflow:add-trino-support

That shows a lot of commits authored by us and committed by you - it looks like a git pull --rebase or something that went bad and resulted in commit-wise conflict resolution and updates. Otherwise those commits shouldn't show up at all. If you look at them on your branch vs main the commit hashes are different, so as far as git is concerned they're new.

Rewriting history is probably in order here, but I don't want to just do that on your working branch. When I do the full diff read-through I'll see if that's something I can do just before we merge. It's possible squash and merge will solve this, depending on what changes crept in during the rebase/merge process.